packages/qwik-city/src/buildtime/build-pages-rewrited.unit.ts

```diff
@@ -228,3 +228,39 @@ testWithDuplicatedRoutes(
     assert.equal(r.pathname, '/produkt/');
   }
 );
+
+const testSameRoutes = testAppSuite('ssss', {
+  rewriteRoutes: [
+    {
+      prefix: undefined,
+      paths: {},
+    },
+    {
+      prefix: undefined,
+      paths: {
+        produkt: 'produkt',
+      },
+    },
+    {
+      prefix: undefined,
+      paths: {
+        booking: 'reservation',
+      },
+    },
+    {
+      prefix: undefined,
+      paths: {
+        booking: 'buchen',
+      },
+    },
+  ],
+});
+
+testSameRoutes(
+  'Issue #6799: Bug while using rewrite routes pointing to the same file',
+  ({ assertRoute }) => {
+    const r = assertRoute('/produkt/');
```

Comment:
So the duplicate routes are for "booking", right? Shouldn't you test that then?

Reply:
Yes I did. And also I tested that project, and everything is fine

Reply:
I mean, in this test :sweat_smile:

Reply:
>So the duplicate routes are for "booking"
yes. and I tested it.

and put my code into project, and it fine

Reply:
Sorry I really don't understand, you have a test here for `/produkt/` and you're not testing the two rules for `booking`

Reply:
Sorry I got it wrong. 
I fixed it. please check

Reply:
this error because is `routeToPush` will be added more than one when `paths` didn't be found. also when paths is empty

Reply:
if I add a `paths` and the matched file was't exist, and `dynamic route` will be putted into list